sql/wsrep_mysqld.cc

```diff
@@ -1653,9 +1656,100 @@ wsrep::key_array wsrep_prepare_keys_for_toi(const char *db, const char *table,
                                               wsrep::key::exclusive));
     }
   }
+  if (fk_tables && !fk_tables->empty()) {
+    ret.insert(ret.end(), fk_tables->begin(), fk_tables->end());
+  }
   return ret;
 }
 
+/*
+ * Collect wsrep keys corresponding to tables that are referencing given table
+ */
+static bool collect_fk_children(const dd::Table *table_def,
+                                wsrep::key_array *keys) {
+  for (const dd::Foreign_key_parent *fk : table_def->foreign_key_parents()) {
+    WSREP_INFO("(c) appended fkey %s", fk->child_table_name().c_str());
```

Comment:
This doesn't seem like something we would want to print on info level?

Reply:
fixed (log removed)